### PR TITLE
Fix installation of powermgmt-base

### LINKED ISSUE #35: Fix installation of powermgmt-base package
See https://github.com/hifis-net/ansible-role-unattended-upgrades/blob/v1.12.2/tasks/unattended-upgrades.yml#L15

### PR DESCRIPTION
**Pull request by Normo**
* Closes #35

### CODE REVIEW

File: molecule/default/verify.yml [hunk 1]
```diff
@@ -7,6 +7,10 @@
       register: "aptconfig"
       changed_when: false
 
+    - name: "Print apt-config variables"
+      debug:
+        msg: "{{ aptconfig.stdout }}"
```
Comment (tobiashuste) [inline molecule/default/verify.yml:12]: What is this useful for?

**Review by tobiashuste — CHANGES_REQUESTED** [review]
This looks good from my side, thanks!

The verify stage of Molecule is failing. Could it be that this line

https://github.com/hifis-net/ansible-role-unattended-upgrades/blob/98170c30eb0e05142fe11f17f0694c08ade41da8/templates/unattended-upgrades.j2#L164

needs to be changed to

```
Unattended-Upgrade::OnlyOnACPower "{{ unattended_only_on_ac_power | to_json }}";
```

to make it work. As I'm on train I couldn't test my assumption locally.

File: molecule/default/verify.yml (hunk 1 above)
Comment (Normo) [reply molecule/default/verify.yml:12]: This was part of the debug commit a8fbc201a15d0002283853c955f24592eff28266 which I'll revert.

**Review by tobiashuste — APPROVED** [review]
Thanks for spotting and fixing this, @Normo :rocket:

**Pull request merged**